### PR TITLE
[devicelab][windows] Install TigerVNC server instead of TightVNC

### PR DESCRIPTION
**Pull request by digiter**
And a few improvements:
* Moves the provision script to a new directory.
* Clone cocoon repository directly in the script.
* Removes starting a Cocoon agent on startup.

Fix https://github.com/flutter/flutter/issues/44275

### CODE REVIEW

File: devicelab/windows_android.ps1 [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2016, the Flutter project authors.  Please see the AUTHORS file
```
Comment (godofredoc) [inline devicelab/windows_android.ps1]: 2019?

File: devicelab/windows_android.ps1 [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+choco install -y dart-sdk
+choco install -y android-sdk
+choco install -y sysinternals
+
+# Installs a VNC server and a SSH server for remote management
+choco install -y tigervnc
+choco install -y openssh
+
+# Installs convenient tools
+choco install -y vscode
```
Comment (godofredoc) [inline devicelab/windows_android.ps1]: Do we actually need this? I'd prefer something that can be used over ssh, e.g. vim

File: devicelab/windows_android.ps1 [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+choco install -y GoogleChrome
+
```
Comment (godofredoc) [inline devicelab/windows_android.ps1]: Can we also add "Set-ExecutionPolicy Unrestricted"

File: devicelab/windows_android.ps1 (hunk 1 above)
Comment (digiter) [reply devicelab/windows_android.ps1]: Done.

File: devicelab/windows_android.ps1 (hunk 2 above)
Comment (digiter) [reply devicelab/windows_android.ps1]: No problem, added vim too.

File: devicelab/windows_android.ps1 (hunk 3 above)
Comment (digiter) [reply devicelab/windows_android.ps1]: What's the benefit of doing that?

Comment (godofredoc) [reply devicelab/windows_android.ps1]: When we are running powershell scripts over ssh some of the will fail is the execution policy is not set correctly. I didn't try any of the other more restrictive policies just went with the most permissive one.

Comment (digiter) [reply devicelab/windows_android.ps1]: SG, done.

**Pull request merged**